rllib/tuned_examples/bc/cartpole_recording.py

```diff
@@ -1,4 +1,9 @@
+import ray
```

Comment:
This should be a regular example script, not a `tuned_example`.
* Let's put this under `examples/offline_rl`.
* Add a docstring at the top, standard for example scripts: what this does, how to run, what to expect, blabla
* Add the the CI (BUILD)!